performance-tests/workbench-assets/src/main/java/.gitignore

```diff
@@ -0,0 +1 @@
+# empty
```

Comment:
you tell me :) this is generated by Business Central automatically - I have just copied the project from there. Since it is actually not necessary I will remove it in the next commit. :)